SourceX/miniwin/misc.cpp

```diff
@@ -182,6 +186,9 @@ bool SpawnWindow(LPCSTR lpWindowName, int nWidth, int nHeight)
 		ErrSdl();
 	}
 
+
+#if !defined(__3DS__)

```

Comment:
This `ifdef` shouldn't be necessary

Reply:
This one actually is needed. Haven't checked which subsystem fails, but it fails during runtime.